src/internal/deobfuscateRenderer.ts

```diff
@@ -6,9 +6,5 @@ export const deobfuscateRenderer = (
   renderer: string,
   isMobileTier: boolean
 ) => {
-  if (renderer === 'apple gpu') {
-    return deobfuscateAppleGPU(gl, renderer, isMobileTier);
-  } else {
-    return [renderer];
-  }
+  return renderer === 'apple gpu' ? deobfuscateAppleGPU(gl, renderer, isMobileTier) : [renderer];
```

Comment:
I had originally formatted like it is now because I wanted to make it future compatible. Seeing it is such a small change to revert it this is fine.